met-web/src/components/engagement/form/EngagementWidgets/WhoIsListening/WhoIsListeningContext.tsx

```diff
@@ -57,7 +59,7 @@ export const WhoIsListeningProvider = ({ children }: { children: JSX.Element | J
                 return Promise.resolve([]);
             }
             setLoadingContacts(true);
-            const loadedContacts = await getContacts();
+            const loadedContacts = await getContactsTrigger(undefined, false).unwrap();
```

Comment:
I am a bit weirded out about passing undefined to the method parameter, hopefully it’s for s good reason?

Reply:
yeah you're right. When you define any parameters that the method should take like  (widgetId, engagementId, something) the trigger function will have its first parameter defined as type void. I necessarily needed the second parameter to be false to ignore the cache in this case but couldn't find a value that it accepts other than undefined for the first parameter. Happy to do some more research around this
![image](https://user-images.githubusercontent.com/91914654/207136981-d79efeb2-d3ff-450a-a649-feecb67c7915.png)